### PR TITLE
BZ1772649: Add OCS 4.2 Info to OCP 4.2+ Docs

### PR DESCRIPTION
**Pull request by bobfuru**
[BZ 1772649](https://bugzilla.redhat.com/show_bug.cgi?id=1772649) Adds OCS 4.2 assembly to OCP 4.2+ documentation (pending 12/19/19 GA of OCS 4.2). Also mentioned here: https://github.com/openshift/openshift-docs/issues/17796#issuecomment-566194565

### CODE REVIEW

**Comment by bobfuru** [comment]
@xltian This is the same content as what is in the Google doc. It has been reviewed and approved by Eng and PM. PTAL - thanks!

**Review by RazTamir — APPROVED** [review]

**Comment by vikram-redhat** [comment]
@bobfuru GA has been postponed till Jan 15th.  Do not merge till then.

**Comment by liangxia** [comment]
The title tells OCP 4.2+, while the labels contain 4.1
Are we going to include the content in OCP 4.1 or not ?

**Comment by bobfuru** [comment]
> The title tells OCP 4.2+, while the labels contain 4.1
> Are we going to include the content in OCP 4.1 or not ?

@liangxia No, not in 4.1. Removed label.
Can I get a LGTM? Thanks.

**Pull request merged**
